src/main/java/uk/gov/hmcts/ccd/data/casedataaccesscontrol/RoleAssignmentAttributesResource.java

```diff
@@ -16,9 +17,9 @@
 public class RoleAssignmentAttributesResource implements Serializable {
```

Comment:
a Javadoc explaining why we map these as Optional and the deserialisation rules would be useful